test/unit/DateTime/Clock/RealtimeClockTest.php

```diff
@@ -8,34 +8,42 @@
 
 
 use Ingenerator\PHPUtils\DateTime\Clock\RealtimeClock;
+use PHPUnit\Framework\TestCase;
 
-class RealtimeClockTest extends \PHPUnit_Framework_TestCase
+class RealtimeClockTest extends TestCase
 {
 
+    protected $newSubject;
+
+    public function setUp()
+    {
+        $this->newSubject = new RealtimeClock;
+    }
+
     public function test_it_is_initialisable()
     {
-        $this->assertInstanceOf(RealtimeClock::class, $this->newSubject());
+        $this->assertInstanceOf(RealtimeClock::class, $this->newSubject);
     }
 
     public function test_it_returns_current_time()
     {
-        $time = $this->newSubject()->getDateTime();
+        $time = $this->newSubject->getDateTime();
         $this->assertInstanceOf(\DateTimeImmutable::class, $time);
         // Allow for time changing during the test
         $this->assertEquals(new \DateTimeImmutable, $time, 'Should be roughly the real time', 1);
     }
 
     public function test_it_returns_microtime()
     {
-        $mt = $this->newSubject()->getMicrotime();
+        $mt = $this->newSubject->getMicrotime();
         $this->assertInternalType('float', $mt);
         $this->assertEquals(microtime(TRUE), $mt, 'Should be roughly the real microtime', 0.2);
     }
 
     public function test_it_sleeps()
     {
         $start   = microtime(TRUE);
-        $subject = $this->newSubject();
+        $subject = $this->newSubject;
         $subject->usleep(500);
         $end = microtime(TRUE);
         $this->assertEquals(
```

Comment:
It should always sleep at least 500 microseconds in every environment, so maybe just split this to be more tolerant if it sleeps a bit longer e.g.
```php
$slept_for = $end - $start;
$this->assertGreaterThanOrEqual(0.0005, $slept_for, 'Should always sleep at least the minimum time');
$this->assertLessThan(0.004, $slept_for, 'May sleep a bit longer if running slow');
```
I think that should cover most build environments, while being fairly confident we're passing the right value to usleep.